src/embedding_generator.py

```
@@ -457,9 +459,9 @@ def embed_chunks(self, chunks: List, layer: int) -> np.ndarray:
 
         logger.info(f"Embedding Layer {layer}: {len(chunks)} chunks")
 
-        # Extract texts - always use 'content' field
-        # For Layer 3, this includes SAC summary (58% DRM reduction!)
-        texts = [chunk.content for chunk in chunks]
+        # Extract texts - concatenate content + raw_content
+        # This gives raw text 2× weight in embedding
```

Ruff (0.14.5)

Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)